src/pages/Settings/TwoStepVerificationPage.vue

```diff
@@ -10,7 +10,7 @@
 		</div>
 		<div class="row">
 			<div class="column small-12 large-8">
-				<kv-settings-card title="How would you like your verification code?">
```

Comment:
Would recommend setting this to `:title="cardTitle"` and removing the h3 with cardTitle below. I may be mistaken, but we should have an overarching `<h2>` for this card which this title prop provides us.

Reply:
So the issue with using the card title, is the location of the card title when a user has MFA enabled. In that case, the card title needs to move below the the newly shown "2-step verification is turned on" and "security method(s)" pieces of the page. 
![MFA_Step_2_v1_Sept_14_2020_RubyPark__–_Figma-3](https://user-images.githubusercontent.com/1521381/99609798-6d101a00-29c5-11eb-8043-34c8b26a61b3.jpg)